helpdesk_mgmt_activity/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import ast
+
+from odoo import _, api, fields, models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    can_create_activity = fields.Boolean(related="team_id.is_set_activity")
+    res_model = fields.Char(string="Source Document Model", index=True)
+    res_id = fields.Integer(string="Source Document", index=True)
+
+    record_ref = fields.Reference(
+        selection="_referenceable_models",
+        compute="_compute_record_ref",
+        inverse="_inverse_record_ref",
+        string="Source Record",
+    )
+    source_activity_type_id = fields.Many2one(comodel_name="mail.activity.type")
+    date_deadline = fields.Date(string="Due Date", default=fields.Date.context_today)
```

Comment:
why are you use `context_today` format?

Reply:
> why are you use `context_today` format?

I used the same format like date_deadline field format in mail.activity model.

Reply:
@geomer198 in the case of an activity, it is assigned to a specific user and it makes sense to take into account the context of the time zone of this user, in the case of a ticket I think it is unnecessary, different users with different time zone can work with a ticket. so, we need use `fields.Date.today()`